core/daemon/src/main/resources/META-INF/opennms/applicationContext-daemon.xml

```diff
@@ -29,13 +29,26 @@
   <onmsgi:reference id="eventProxy" interface="org.opennms.netmgt.events.api.EventProxy"/>
   <onmsgi:reference id="eventSubscriptionService" interface="org.opennms.netmgt.events.api.EventSubscriptionService"/>
 
+  <!-- TODO: Remove this bean after we stop using serialized object messages -->
```

Comment:
I don't think that we do now but the original commits in this branch pre-dated that work. I'll track removing the ```org.apache.activemq.SERIALIZABLE_PACKAGES``` system properties in a new issue.